easybuild/easyconfigs/g/GTS/GTS-0.7.6-foss-2016a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'GTS'
+version = '0.7.6'
+
+homepage = 'http://gts.sourceforge.net/'
+description = """GTS stands for the GNU Triangulated Surface Library.
+ It is an Open Source Free Software Library intended to provide a set of useful
+ functions to deal with 3D surfaces meshed with interconnected triangles."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('GLib', '2.47.5'),
```

Comment:
there's an update for GLib, v2.48.0, I'm working on an easyconfig for it, PR coming up...